tests/test_import/module_S3.R

```diff
@@ -0,0 +1,20 @@
+
+# Testing two cases:
+# a) method for new generics
+# b) methods for existing generics
+test_fun <- function(x){
+    UseMethod("test_fun", x)
+}
+
```

Comment:
Document these functions. Does not need to be full roxydocs, but a line on what it does, given that there is not that much external documentation of this feature.